crowdfunding_compassion/models/crowdfunding_participant.py

```diff
@@ -84,3 +85,18 @@ def _compute_sponsorships(self):
     def _compute_number_sponsorships_reached(self):
         for participant in self:
             participant.number_sponsorships_reached = len(participant.sponsorship_ids)
+
+    @api.multi
+    def _compute_profile_photo_url(self):
+        domain = self.env['website'].get_current_website()._get_http_domain()
```

Comment:
Did you test this in private tab? I wonder if an unregistered user has security access to `get_current_website` otherwise you should add `sudo()` Unless the sudo is already handled upper in the code each time the participant are accessed from the website. In that case you could as well remove the `sudo` calls below.

Reply:
Yes it's working so they seem to have access too. The sudo below are required however, I had rights error messages without them, logged in and logged out.

Reply:
Yes it's working so they seem to have access too. The sudo below are required however, I had rights error messages without them, logged in and logged out.